### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kdorichev**
Scripts diff_nb_script.py, notebook2script.py and script2notebook.py described.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,5 +32,6 @@ for i in {0,1,2}*.ipynb; do sleep 1; python run_notebook.py --fn $i & done
 
 After you clone this repository, please run `tools/run-after-git-clone` in your terminal. This sets up git hooks, which clean up the notebooks to remove the extraneous stuff stored in the notebooks (e.g. which cells you ran) which causes unnecessary merge conflicts.
 
-There are various `.py` scripts in the `dev` directory which you should look at (and we should document!)
-
+Before submitting a PR, check that the local library and notebooks match. The script `diff_nb_script.py` can let you know if there is a difference between the local library and the notebooks.
```
Comment (amaarora) [inline README.md]: Great idea to add this to readme! :) 

I would just add this one line too quoted from Sylvain's post [here](https://forums.fast.ai/t/submitting-pr-to-fastai-v2/54146/9?u=arora_aman) "If it doesn’t print anything, you are good to go."

Comment (amaarora) [inline README.md]: Great idea to add this to readme! :) 

I would just add this one line too quoted from Sylvain's post [here](https://forums.fast.ai/t/submitting-pr-to-fastai-v2/54146/9?u=arora_aman) "If it doesn’t print anything, you are good to go."

Comment (kdorichev) [reply README.md]: Thanks for the suggestion, @amaarora. I've considered keeping that line, but chose to eliminate it to keep it short.

Comment (amaarora) [reply README.md]: Sounds good :)

**Comment by jph00** [comment]
Thanks!

**Pull request merged**